.storybook/main.js

```diff
@@ -3,7 +3,6 @@ module.exports = {
   addons: [
     "@storybook/addon-docs",
     "@storybook/addon-controls",
-    "@storybook/addon-queryparams",
```

Comment:
It was removed again because it was not needed anyways in this context.
But meanwhile Sine added it with the demo-modal PR.
So yes, it should not be removed.
I'll remove that part

Reply:
Or rather reintroduce :)

Reply:
Såen